app/uk/gov/hmrc/customsservicestatusfrontend/models/ServiceStatus.scala

```diff
@@ -18,31 +18,33 @@ package uk.gov.hmrc.customsservicestatusfrontend.models
 
 import play.api.libs.json.*
 import play.api.libs.json.Json.WithDefaultValues
+import uk.gov.hmrc.customsservicestatusfrontend.utils.EqualUtils.AnyOps
 
 import java.time.Instant
 
-sealed trait State {
+enum State {
+  case Available
+  case Unavailable
+  case Unknown
+
   val value: String = toString
 }
 
 object State {
-  case object AVAILABLE extends State
-  case object UNAVAILABLE extends State
-  case object UNKNOWN extends State
 
-  val values: Seq[State] = Seq(AVAILABLE, UNAVAILABLE, UNKNOWN)
+  def apply(value: String): Option[State] =
+    values.find(_.value === value)
```

Comment:
This method is also unused because you're using the built-in `valueOf` method and catching the exception instead. So think this can be removed.